lib/atoms/button/example.html

```diff
@@ -0,0 +1 @@
+<button id="{{id}}" class="{{class}}">Click me!</button>
```

Comment:
We should add ng-click to this example. Also "Click me!" should be something like {{text}} in order to change its value from the patterns configuration.